docs/source/design/github_actions.rst

```diff
@@ -0,0 +1,84 @@
+.. -*- mode: rst -*-
+.. vi: set ft=rst sts=4 ts=4 sw=4 et tw=79:
+
+.. _chap_design_github_action:
+
+*******************************
+GitHub Action
+*******************************
+
+.. topic:: Specification scope and status
+
+   This specification describes a proposed interface to a DataLad GitHub Action.
+
+The purpose of the DataLad GitHub Action is to support CI testing with DataLad datasets
+by making it easy to install ``datalad`` and ``get`` data from the dataset.
+
+
+Example Usage
+=============
+
+Dataset installed at ``${GITHUB_WORKSPACE}/studyforrest-data-phase2``,
+``get``'s all the data::
+
+    - uses: datalad/datalad-action@master
+      with:
+        datasets:
+          - source: https://github.com/psychoinformatics-de/studyforrest-data-phase2
+
+Specify advanced options::
+
+    - name: Download testing data
+      uses: datalad/datalad-action@master
+      with:
+        datalad_version: ^0.15.5
+        datasets:
+          - source: https://github.com/psychoinformatics-de/studyforrest-data-phase2
```

Comment:
I believe that actions could add to GITHUB_ENV file thus augmenting PATH. Let's add option 

```
add_datalad_to_path: bool
```

which should then add its miniconda-installed datalad path into PATH so subsequent actions could make use of the `datalad` command.  The fear is that conda env might interfere with the rest of the setup.  We might want to provide only some "datalad" shim which would then export PATH to conda for its duration.

Than we can allow for arbitrary desired flexibility of `datalad get` invocations in subsequent steps.

Reply:
Good idea, added!